.github/workflows/kvrocks.yaml

```diff
@@ -338,11 +346,15 @@ jobs:
         with:
           go-version-file: 'tests/gocase/go.mod'
 
-      - name: Build Kvrocks
+      - name: Build Kvrocks for CentOS
         if: ${{ startsWith(matrix.image, 'centos') }}
         run: |
           source /opt/rh/devtoolset-11/enable
```

Comment:
@PragmaTwice is it possible we move this line to the setup stage so that all "Build Kvrocks" share the same script?

Reply:
We can have some try, but I think it is not so easy.

Reply:
> @PragmaTwice is it possible we move this line to the setup stage so that all "Build Kvrocks" share the same script?

I found a way to do this, maybe not elegant.  :)